LayoutTests/compositing/rtl/rtl-fixed-clipping-layer-expected.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <style>
+        html {
+          direction: rtl;
+        }
+
+        body {
+          margin: 0;
+          width: 1550px;
+          overflow:hidden;
```

Comment:
I suggest either making this an LTR reference, or making it not-hidden; seems excessive to do both.

(Also it's a bit weird to propagate direction from html and overflow from body... but it's not wrong. :)